.github/workflows/helm.yml

```diff
@@ -78,8 +78,27 @@ jobs:
         pip3 install --user -r tests/python/requirements.txt
 
     - name: REST API and SDK tests
+      id: run_tests
       # We don't have external services in Helm tests, so we ignore corresponding cases
       # They are still tested without Helm
       run: |
         kubectl cp tests/mounted_file_share/images $(kubectl get pods -l component=server -o jsonpath='{.items[0].metadata.name}'):/home/django/share
         pytest --timeout 30 --platform=kube -m "not with_external_services" tests/python
+
+    - name: Creating a log file from cvat containers
+      if: failure() && steps.run_tests.conclusion == 'failure'
+      env:
+        LOGS_DIR: "${{ github.workspace }}/testing"
+      run: |
+          mkdir ${LOGS_DIR}
+          for container_name in "test_cvat_server_1" "test_cvat_worker_export_1" "test_cvat_worker_import_1" "test_cvat_opa_1";
+          do
+            docker logs ${container_name} &> ${LOGS_DIR}/${container_name}.log
+          done
+
+    - name: Uploading "cvat" container logs as an artifact
+      if: failure() && steps.run_tests.conclusion == 'failure'
+      uses: actions/upload-artifact@v3.1.2
+      with:
+        name: rest_api_container_logs
+        path: "${{ github.workspace }}/testing"
```

Comment:
I'm not sure these changes are relevant to the PR, could you please check?

Reply:
I've added it because I didn't understand why the helm tests failed just by tests log (but after adding it helm tests passed). 
I can extract this to another PR, but I guess that we should have the assets with server logs for helm tests too.

Reply:
@zhiltsov-max, I've removed it because it was wrong and  I just saw that this has already been implemented in [PR](https://github.com/opencv/cvat/pull/6013)